materialize-databricks/driver.go

```diff
@@ -277,6 +279,12 @@ func (c client) PreReqs(ctx context.Context, ep *sql.Endpoint) *sql.PrereqErr {
 		case databricksSql.StateStopping:
 			errs.Err(fmt.Errorf("The selected SQL Warehouse is stopping, please start the SQL warehouse and try again."))
 		}
+
+		if res.AutoStopMins >= 15 {
+			log.Warn(fmt.Sprintf("Auto-stop is configured to be %d minutes for this warehouse, disabling update delay. To save costs you can reduce the auto-stop idle configuration and tune the update delay config of this connector. See docs for more information: https://go.estuary.dev/materialize-databricks", res.AutoStopMins))
+
+			cfg.Advanced.UpdateDelay = "0s"
```

Comment:
Since we're setting this in `NewTransactor` I don't think we need to do it here too